core/templates/dev/head/filters/string-utility-filters/camel-case-to-hyphens.filter.spec.ts

```diff
@@ -16,24 +16,24 @@
  * @fileoverview Tests for CamelCaseToHyphens filter for Oppia.
  */
 
-require('filters/string-utility-filters/camel-case-to-hyphens.filter.ts');
```

Comment:
Where's the removal for this filter?

Reply:
I haven't removed that since the downgraded version is still used in many services. After the migration is done we will delete all the filters.

Reply:
okay, in that case, please go ahead and make an issue (assign it to me and/or yourself) detailing the removal of these filters

Reply:
Done https://github.com/oppia/oppia/issues/7716

Reply:
thanks!

Reply:
@bansalnitish So there aren't any tests for existing filters?

Reply:
> I haven't removed that since the downgraded version is still used in many services.

Is there any issue with changing the services such that it uses the new filters?

Reply:
The existing filters will be removed soon. The behaviour of both filters is identical so I don't think we need to have same tests for filters. 

Umm, I'm not sure (haven't read that anywhere) -- we are moving in dependency order and we will cover all services. My plan is to do like -- if we are migrating a service X, and X uses filters then replace those with new filers.

Reply:
> @bansalnitish So there aren't any tests for existing filters?

This way the filter in use doesn't have any tests, right? IF that's the case then I think th test should be for the old filters as well or we should completely remove the old filters in the same PR!

@kevinlee12, any thoughts?

Reply:
@kevinlee12 @bansalnitish, any thoughts on this?

Reply:
Yes, you're right the existing filter does not have any tests. We can add the same file if you say but I guess we don't need one right now due to two reasons:
1. The old filters will go away soon.
2. The old and new filters have exactly the same functionality -- we would have two test files almost having the same code and testing the same code.

Reply:
>  we would have two test files almost having the same code and testing the same code.

Yeah, that why I was asked to either remove one completely or have a test for them both (I would prefer the former one). Even the test for the filter which is in use is more important than which isn't in use!

Reply:
@bansalnitish, is this resolved?

Reply:
Both are in use @DubeySandeep. We cannot remove old filters since they are used too. 

You want tests for both of them then?

Reply:
Added the test file back for the filter too!